packages/react-native-tab-view/src/PlatformPressable.tsx

```diff
@@ -30,16 +30,16 @@ export function PlatformPressable({
   ...rest
 }: Props) {
   const handlePress = (e: GestureResponderEvent) => {
-    // @ts-expect-error: these properties exist on web, but not in React Native
-    const hasModifierKey = e.metaKey || e.altKey || e.ctrlKey || e.shiftKey; // ignore clicks with modifier keys
```

Comment:
Since these values are only ever checked inside the first if-condition, move all of them inside that if block.